index.html

```diff
@@ -233,9 +233,9 @@ <h2>Languajes
         <h1>Contact me</h1>
         <p>If you have an application you are interested in developing, a feature 
             that you need built or a project that needs coding. I’d love to help with it </p>
-            <input type="text" id="form-name" placeholder="Yeremias">
-            <input type="email" id="form-email" placeholder="natajayanj14@gmail.com">
-            <input type="text" id="form-message" placeholder="Write your message here">
+            <input type="text" id="form-name" placeholder="Enter name" maxlength="30">
+            <input type="email" id="form-email" placeholder="Enter email">
+            <input type="text" id="form-message" placeholder="Write your message here" maxlength="500">
         <div class="form-button">
             <button>Get It Touch</button>
```

Comment:
- [ ] Kindly implement the button interactions as requested by the previous reviewer. Click [here](https://www.w3schools.com/howto/howto_css_animate_buttons.asp) to read more on how to achieve that with CSS.